tests/main/interfaces-openvswitch/task.yaml

```diff
@@ -15,8 +15,14 @@ details: |
     list and deletion.
 
 prepare: |
+    . "$TESTSLIB/pkgdb.sh"
+
     echo "Given openvswitch is installed"
-    apt install -y --no-install-recommends openvswitch-switch
```

Comment:
It might be better to be more generic: `--no-weak-deps`, which for Debian could be `--no-install-recommends`. For openSUSE, it would be `--no-recommends`, and for Fedora, it'd be `--setopt=install_weak_deps=False`